Source/diablo.cpp

```diff
@@ -1027,6 +1027,9 @@ void SetApplicationVersions()
 
 void CheckArchivesUpToDate()
 {
+#if defined(__amigaos__)

```

Comment:
Why doesn't this work on Amiga?

Reply:
Actually it's works. I wasn't aware that assets v3 are needed.